main.go

```diff
@@ -66,6 +72,18 @@ func main() {
 	// new
 	rootCmd.AddCommand(new.CreateCommand())
 
+	// initialize
 	cobra.OnInitialize(root.InitLogger, root.InitConfig)
-	cobra.CheckErr(rootCmd.ExecuteContext(ctx))
+
+	// instrument & execute called command
+	ctx, childSpan := telemetry.NewSpan(ctx, calledCommand)
+	err := rootCmd.ExecuteContext(ctx)
+	telemetry.RecordSpanError(childSpan, err)
+	telemetry.EndSpan(childSpan)
+
+	// Send all events
+	telemetry.ShutDown(ctx, traceProvider, parentSpan)
```

Comment:
There was some talk of logging the data  we send before we sent it, would this happen here? What would it look like?